File: assignment 10 - C# practice problems/24.cs
//Write a c# function to get the amount of days of a year. 

using System;
namespace MyNameSpace
{
    class Program24
    {
        public static void AmountOfDaysInYearDriverFunction()
        {


            Console.WriteLine("Enter year in yyyy format: ");
            var dateInput1 = Console.ReadLine();
            var year = int.Parse(dateInput1);

            if (isLeapYear(year))
            {
                Console.WriteLine("366 days");
            }
            else
            {
                Console.WriteLine("365 days");
            }
        }

        static bool isLeapYear(int yearParam)
        {
            if (yearParam % 400 == 0)
            {
                return true;
            }

            if (yearParam % 100 == 0)
            {
                return false;
            }

            if (yearParam % 4 == 0)
            {
                return true;
            }

            return false;
        }
    }
}

File: assignment 10 - C# practice problems/js/24.js
// function isLeapYear(year) {
//     if (year % 4 === 0) {
//         if (year % 100 === 0) {
//             if (year % 400 === 0) {
//                 return true;
//             } else {
//                 return false;
//             }
//         } else {
//             return true;
//         }
//     } else {
//         return false;
//     }
// }

//Write a JavaScript function to get the amount of days of a year. 

function isLeapYear(yearParam) {
    if (yearParam % 400 == 0) {
        return true;
    }

    if (yearParam % 100 == 0) {
        return false;
    }

    if (yearParam % 4 == 0) {
        return true;
    }

    return false;
}

function isLeapYearDriverFunction() {
    let yearInput = prompt("enter year in yyyy");

    if (isLeapYear(yearInput)) {
        alert("366 days");
    } else {
        alert("365 days");
    }
}

isLeapYearDriverFunction();